client/my-sites/plugins/categories/README.md

```diff
@@ -0,0 +1,15 @@
+# Categories
+
+This component is used to render the discover section of the plugin marketplace.
+
+## How to use
+
+```jsx
+import Discover from 'calypso/my-sites/plugins/discover';
```

Comment:
nitpick: shouldn't this be named `Categories` and the path be `'calypso/my-sites/plugins/categories'`

Reply:
Yes, was going to call it discover but renamed it to be more obvious

```suggestion
import Discover from 'calypso/my-sites/plugins/categories';
```